regulations/static/regulations/css/scss/partials/_footer.scss

```diff
@@ -112,7 +132,7 @@ footer {
 
     // divide to strip unit (px)
     // https://css-tricks.com/snippets/sass/strip-unit-function/
-    @include custom-max($mobile-max / 1px) {
+    @include custom-max($tablet-max / 1px) {
```

Comment:
The tablet comp shows the footer being three columns at widths 768px and above:

https://www.figma.com/file/wEaWv7FFtYluyMaJ4qTRsp/eRegs-Full-Responsive-Experience?node-id=429%3A163

Reply:
@PhilR8 I adjusted to break it sooner because It was a bit awkward for smaller tablets once live. iPad Pros will have the 3-column still.

Reply:
Sounds good!